catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/GetRecordsResponseConverter.java

```diff
@@ -102,9 +102,16 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
           reader.moveDown(); // move down to the <csw:Record> tag
           String name = reader.getNodeName();
           LOGGER.debug("node name = {}", name);
-          Metacard metacard =
-              (Metacard) context.convertAnother(null, MetacardImpl.class, transformProvider);
-          metacards.add(metacard);
+
+          try {
+            Metacard metacard =
+                (Metacard) context.convertAnother(null, MetacardImpl.class, transformProvider);
+            metacards.add(metacard);
+          } catch (ConversionException e) {
+            LOGGER.warn(
+                "Failed to convert result to a metacard. Ignoring this resource because: " + e);
+            cswRecords.setNumberOfRecordsMatched(cswRecords.getNumberOfRecordsMatched() - 1);
```

Comment:
❓ What's the behavior if we _don't_ do this? It'd be nice to use this as a way of showing the CSW source that there are other records that didn't make it all the way to the CSW source due to some exception along the way, but I don't know exactly how the CSW source handles that.

Reply:
The purpose of this line was to stop duplicating the log message, as well as saying how many results were actually returned in Intrigue. For example, if 4 resources were present but one failed to transform, it would say there are 4 resources, but only show the three successful ones. I assumed that could be confusing to the user, but is that what we might want, considering there will be more information about it in the logs?

Reply:
So, without this line, say 1 out of 4 metacards fails to transform. In Intrigue, would it display the 3 metacards but say "4 results"? I guess what I'm asking is if the "numberOfRecordsMatched" bubbles up all the way up to Intrigue?

Reply:
Correct

Reply:
Okay. Well my opinion would be to have it say "4 results" even though only 3 are displayed, but I'd like to hear the opinions of other reviewers. It'd also be good to get feedback from the user who reported this bug.

Reply:
After talking to other people such as the person who reported the bug as well as someone who interacts with users, it seems people are leaning towards having it say "3 results" when there are 3 displayed. Marking this conversation as resolved for now, but feel free to open up for further discussion.